src/components/result/index.jsx

```diff
@@ -19,13 +32,64 @@ export default class ListResults extends Component {
   }
 
   render() {
-    const job = this.state.job;
+    const job = this.state.job || {};
     if (this.state.loading) {
       return <div className="loader" />;
     }
+    const {
+      createdAt,
+      updatedAt,
+      status,
+      jobId,
+      map,
+      reduce,
+      url,
+      jobName
+    } = job;
     return (
-      <div className="result">
-        <pre>{JSON.stringify(job, null, 2)}</pre>
+      <div className="result__root">
+        <div className="body">
+          <div className="header">
+            <div>
+              <div className="jobName">{jobName}</div>
+              <div className="subHeader">
+                <div className="label">Created On:</div>
+                <div className="value">
+                  {new Date(createdAt).toDateString()}
```

Comment:
This doesn't include a timestamp - how about using `toLocaleString` instead?

Reply:
Good call. just pushed another update with that change